psqlgraph/query.py

```diff
@@ -25,7 +22,7 @@ def __init__(self, entites, session=None, package_namespace=None):
     def _iterable(self, val):
         if hasattr(val, '__iter__') and not isinstance(val, six.string_types):
             return val
-        return val,
+        return val
```

Comment:
it seems to me that this was trying to return a `tuple` to make sure that the `val` becomes `iterable`

Reply:
:facepalm: my bad totally. Thanks for catching this